app/scripts/react-directives/script-editor/scriptEditor.jsx

```
@@ -4,9 +1,12 @@
 import { javascript } from '@codemirror/lang-javascript';
 import { EditorSelection } from '@codemirror/state';
+import CodeMirror from '@uiw/react-codemirror';
+import { observer } from 'mobx-react-lite';
+import React, { useEffect } from 'react';
```

ESLint

[error] 'default' import from 'react' is restricted.

(no-restricted-imports)